src/components/ClassSearch.vue

```diff
@@ -201,6 +202,13 @@ class MathFilter extends Filter {
   }
 }
 
+class BooleanFilter extends Filter {
+  constructor (name, shortName, attributeNames, negated, mode) {
+    var match = (input) => input == !negated;
```

Comment:
is this meant to be `==` or `===`? If it's really `==`, please add `// eslint-disable-line eqeqeq` to this line to placate the linter